source/grid/grid_generator.cc

```diff
@@ -3111,7 +3112,7 @@ namespace GridGenerator
                   = cell->face(i);
 
                 const Point<3> face_center (face->center());
-                if (std::abs((face_center(0)-center(0)) > 1.e-3))
+                if (std::abs(face_center(0)-center(0)) > 1.e-6)
                   {
```

Comment:
> In reality, the tolerance here needs to be relative to inner/
> outer_radius to accommodate the fact that the user can choose how
> large the domain needs to be.

Yes, exactly. This is also true for the other 200 magic numbers within
the library.

What about providing a generic function to test for equality with eps
that correctly takes relative precision and subnormal representation
into account?